wordpress-mobile/WordPress-Android
# Add task to setup dev dependencies

This PR adds a `gradle` task to make it easier to setup some development dependencies, like `bundle`, `fastlane` and the `release-toolkit` that may be useful for credential management, screenshots, releases and so.

# To test:
- If you have it, delete the `vendor` folder in your repo.
- Run `./gradlew prodDeps`.
- Verify that `bundle` installation is checked and that the tool is installed if missing.
- Verify that all the gems are installed. 
- Verify that the correct credentials are applied.  

PR submission checklist:
